src/js/va-letters/containers/ConfirmAddress.jsx

```diff
@@ -1,19 +1,79 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+import { updateAddressConfirmation } from '../actions';
+
+import ProgressButton from '../../common/components/form-elements/ProgressButton';
+
 class ConfirmAddress extends React.Component {
+  constructor(props) {
+    super(props);
+    this.confirmAddress = this.confirmAddress.bind(this);
+  }
+
+  confirmAddress() {
+    this.props.onConfirmationOfAddress(true);
+    this.props.router.push('/download-letters');
+  }
+
   render() {
+    const buttons = this.props.letters.addressConfirmed !== false ?
```

Comment:
Would it be possible to use `src/js/common/components/form-elements/ErrorableRadioButtons.jsx` with the `yesNo` option labels here, and pass in a handler to its `onValueChange` prop that does the same as below? I haven't looked carefully at it, but am guessing it takes care of the accessibility issues you were concerned about.